ChangeLog.d/ecp-light.txt

```diff
@@ -0,0 +1,8 @@
+Features
+   * The ECP module can now be added incrementally into the build by means of 2
+     different symbols: MBEDTLS_ECP_LIGHT and MBEDTLS_ECP_C. The latter, which
```

Comment:
> mention ECP_LIGHT in the ChangeLog, but at the same time do not put it into the mbedtls_config.h file

No, I don't think that's a good idea. If we mention something in the changelog, it's because it's useful to users. Mentioning an option without explaining how to decide whether to turn it on is not useful.